t/ci/RUN_UNIFYFS_CI_STAGE_TESTS.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+# This script runs a full sweep of tests using the
+# stage application to bring data into the UnifyFS file
+# space and then back out.
```

Comment:
I assume that the user has run 001-setup.sh (because that sets up all the directories and stuff) but NOT 002-start-server.sh.  Because this chain of scripts starts the servers several times with different parameters, and then tears them down each time.

Reply:
I put an explanatory comment about this issue.  As far as integrating this into the rest of the CI stuff, once I have a coherent set of scripts that work both on catalyst and summit, let's deal with that.